closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -982,15 +1015,16 @@ public synchronized int doUpdate(byte[] in, int inOff, int inLen, byte[] out,
                     throw new ShortBufferException("Output buffer too small");
                 }
 
-                Objects.checkFromIndexSize(inOff, inLen, in.length);
+                if (in != null) {
```

Comment:
Why isn't there a similar test `(in != null)` in `EngineStreamOnly.doUpdate()`?

Reply:
Yep. Thanks for the catch. Fixed it.